docs/trex_format.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Add Tableau Workbook Formatting
+layout: docs
+---
+
+To help your dashboard extension match the look and feel of the dashboard in which it is placed, you can apply the formatting styles used in the workbook to the HTML elements in your extension. When the text formatting changes in the workbook, the corresponding styles are updated in your extension.
```

Comment:
While we currently only support fonts, could we future proof this sentence by removing "text" from the "When the text formatting changes...". I don't think this would cause confusion when they see the class names below.

Reply:
Good point. I'll correct that.